src/validator.js

```diff
@@ -337,12 +337,19 @@ export class Validator {
         const errors = []
         if (Array.isArray(schema.required)) {
           schema.required.forEach(e => {
+            if (this._checkType('string', value[e]) && value[e].length === 0) {
```

Comment:
check cases of strings when them are required and values ​​change from `stringsWithALength` to " "(empty)